### PR TITLE
removed dependency from 3rd party image

### PR DESCRIPTION
**Pull request by ruslan-safin**
Just removed dependency from 3rd party image (jrei/systemd-debian:12)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,29 @@
-FROM jrei/systemd-debian:12
-RUN apt update && apt install -y sudo wget procps curl systemd iproute2 && rm -rf /var/lib/apt/lists/*
+FROM debian:bookworm
```
Comment (jrcichra) [inline Dockerfile:1]: Maybe we add a comment linking to where the logic originated from?

The repo's license file: https://github.com/j8r/dockerfiles/blob/master/LICENSE#L5

Comment (ruslan-safin) [reply Dockerfile:1]: Hm, doesn't matter, because this was crated from scratch, but if this already authored. That's ok to create link.

**Comment by jrcichra** [comment]
I'll build and test this tonight. Thanks for the PR!

**Review by jrcichra — APPROVED** [review]
Works on my Pi 5.

**Comment by billz** [comment]
Many thanks for the PR. I'll run through a build tonight.

**Review by NL-TCH — APPROVED** [review]

**Comment by billz** [comment]
Tests pass, good to merge. @ruslan-safin thanks!

**Pull request merged**